_includes/releases/v23.1/v23.1.0-alpha.5.md

```diff
@@ -0,0 +1,166 @@
+## v23.1.0-alpha.5
+
+Release Date: March 6, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-1-0-alpha-5-security-updates">Security updates</h3>
+
+- The new cluster setting `server.user_login.downgrade_scram_stored_passwords_to_bcrypt.enabled`, which allows you to migrate passwords from SCRAM to bcrypt during user authentication, defaults to true. If it is true and if `server.user_login.password_encryption` is `crdb-bcrypt`, then during login, the stored hashed password will be migrated from SCRAM to bcrypt. [#97429][#97429]
+
+<h3 id="v23-1-0-alpha-5-general-changes">General changes</h3>
+
+- Users with the `CONTROLJOB` role option can now view jobs owned by admins (ex. via `SHOW JOBS`). [#96382][#96382]
+- Users with the `VIEWJOB` role option can now view all jobs, such as by using the `SHOW JOBS` command. This role can be revoked by granting the `NOVIEWJOB` role. [#96382][#96382]
+
+<h3 id="v23-1-0-alpha-5-{{-site.data.products.enterprise-}}-edition-changes">{{ site.data.products.enterprise }} edition changes</h3>
+
+- Jobs that utilize protected timestamp system (such as `BACKUP`, `CHANGEFEED`, or `IMPORT` now produce metrics that can be monitored to detect cases when a job leaves a stale protected timestamp that will prevent garbage collection from occurring. [#97148][#97148]
+- Changefeeds now automatically expire PTS records for paused jobs if the changefeed is configured with the `gc_protect_expires_after` option. [#97148][#97148]
```

Comment:
What is a PTS record? Can this be spelled out?

Also, link to changefeeds doc.